public/app/features/dashboard/components/PanelEditor/PanelEditor.tsx

```diff
@@ -56,15 +60,17 @@ interface State {
 export class PanelEditor extends PureComponent<Props, State> {
   state: State = {};
 
-  componentDidMount() {
-    const { panel } = this.props;
+  constructor(props: Props) {
+    super(props);
+    const { panel } = props;
     const dirtyPanel = new PanelModel(panel.getSaveModel());
-
-    this.setState({ dirtyPanel });
+    dirtyPanel.setQueryRunner(panel.getQueryRunner().clone());
```

Comment:
Cloning the original panel model's query runner, to avoid updates in the original panel. Any risk with that that you can see @torkelo ?